posthog/temporal/usage_reports/run_usage_reports.py

```diff
@@ -227,6 +211,8 @@ def async_queue_report(p, oid, frd) -> bool:
         )
         pha_client.flush()  # Flush and close the client
 
+        return QueryUsageReportsResult()
```

Comment:
There doesn't seem to be a reason to return anything anymore. Couldn't we just get rid of this?